app/test_split_integer.py

```diff
@@ -2,20 +2,36 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 17
+    number_of_parts = 4
+    result = split_integer(value, number_of_parts)
+    assert sum(result) == value
```

Comment:
Based on clean code recommendations, will be better to do:

```suggestion
    assert sum(split_integer(17, 4)) == 17
```
![image](https://github.com/user-attachments/assets/35ea20d0-e5ac-40b2-9c12-1a1c950d20a5)